robot.py

```diff
@@ -34,10 +42,10 @@ def teleopPeriodic(self):
             self.shooter_mod = 0.2
```

Comment:
More state that should be in the shooter.    If the intent is to provide different speeds to acheive different ranges, then that should be reflected in the naming of the function.

But I'm not sure about what your design is for this.      This works great as a testbed for electrical/mechanical to see how the shooter works at different speed settings.       But if we want precision shooter, the shooter is going to have to eventually use the the encoders in the motor and the PIDController object to spin the motor at a precise RPM.     That would give use proper control of range.     And that is where the real funs begins.

Reply:
I'm actually pretty confused about if this is supposed to be an actual shooter or something to help electrical test their motor myself. Electrical says that they can't test their motor without code, so it would make sense if this was that, but Soren seems to think that this is for the shooter. Either way, Steve wrote this exact control design on the whiteboard, so that's what we implemented. I think he said that it was for electrical so that they could test their motor, but I'm not sure. 

If this is really for the shooter, Soren and I agree that this design is questionable so we'll probably write our own design sometime this week.

Reply:
There are likely two competing objectives.      Electrical needs a test harness, but it doesn't have to suck.   You can still encapsulate all of the shooter state and operations appropriately rather than scattering them throughout the robot code.    But, the other is that we need proper shooter code, which will likely take more time to develop, and may not be that helpful to electrical in the early stages.